patches/0002-Add-crypto-backend-foundation.patch

```diff
@@ -355,10 +355,10 @@ index 00000000000000..e5d7570d6d4363
 +const isRequireFIPS = true
 diff --git a/src/crypto/internal/backend/nobackend.go b/src/crypto/internal/backend/nobackend.go
 new file mode 100644
-index 00000000000000..ad6081552af15d
+index 00000000000000..3698bbac41e5c8
 --- /dev/null
 +++ b/src/crypto/internal/backend/nobackend.go
-@@ -0,0 +1,116 @@
+@@ -0,0 +1,125 @@
 +// Copyright 2017 The Go Authors. All rights reserved.
```

Comment:
I assume by convention we would not update the copyright date?

Reply:
Yep, that's the upstream convention too.